src/cli/llm/infer_type_name.rs

```diff
@@ -149,6 +180,15 @@ impl InferTypeName {
 
         Ok(new_name_mappings.into_iter().map(|(k, v)| (v, k)).collect())
     }
+
+    async fn ask_with_context(
+        &self,
+        system_messages: &[ChatMessage],
+        question: &Question,
+    ) -> Result<Answer> {
+        // Remove the conversion to ChatRequest
+        self.wizard.as_ref().unwrap().ask(question.clone()).await
```

Comment:
our goal was to send system message only once but question itself contains the system message.